### DISCUSSION TITLE
I recieved an email for academic reverification, but the form says I'm already verified and can't submit it

### DISCUSSION THREAD

**Discussion by Ygilany** [community/community#129898] [GitHub Education]
### Hi everyone,

I recieved an email regarding my academic status and that I need to reverify my status. 

Clicking on the link I'm taken to the benefits verification page. but that page says I'm already verified and wouldn't let me proceed with applying

**Comment by Khirasagar** [comment]
Double-Check the Email: Ensure the email you received is legitimate and not a phishing attempt. Verify the sender's address and the content of the email.

**Reply by Ygilany** [reply]
The email is legit
<img width="1023" alt="Screenshot 2024-06-28 at 12 02 27 PM" src="https://github.com/community/community/assets/11262007/171d8a19-9d59-4a1b-98ab-88ebd6328c86">

**Comment by danisoaresl** [comment]
Contact Support: Reach out to the support team of the academic institution or the service you are trying to verify with. Provide them with the details of the issue, including screenshots if possible, and ask for assistance.

**Reply by Ygilany** [reply]
Thank you Dani, the only support for GitHub Education I could find is this post. Please let me know if there's another platform where I can get help

**Comment by Ygilany** [comment]
pumping it up

**Comment by Ygilany** [comment]
Support hasn't gotten back to me in over 2 weeks!!!

**Comment by Ygilany** [comment]
Semester is starting soon and I can't setup my classes

**Comment by Ygilany** [comment]
Folks, I need some one  from GitHub Support to reach back out to me.. I've submitted multiple tickets and posted here but no help!

I need to be able to setup my classrooms before the semester starts.
